app/views/shared/_display_errors.html.slim

```diff
@@ -0,0 +1,7 @@
+-if thing.errors.any?
+  .form-error
+    p #{pluralize(thing.errors.count, "error")} prevented this #{thing.class.to_s.downcase} from being saved
+    ul
+      - thing.errors.full_messages.each do |msg|
```

Comment:
NB: this will correctly iterate through the errors on the specific object, but if you have nested objects with errors (such as if you're using `accepts_nested_attributes_for`) then you might want something with a little more logic.